### DISCUSSION TITLE
Disable Achievements Across GitHub

### DISCUSSION THREAD

**Discussion by andrewtavis** [community/community#18151] [Other Feature Feedback, Questions, & Ideas] [answered]
It'd be great if disabling achievements for one's profile also had the extra option to disable it for the whole website. I get that this feature might be interesting to some, but to others it's a distraction.

Thanks for your all's efforts!

**Comment by bailey** [comment, marked as answer]
Thank you for this feedback @andrewtavis! We'll definitely consider this as we iterate on the controls for Achievements.
